SpringProject/src/main/java/com/gradproject2019/users/service/UserServiceImpl.java

```diff
@@ -77,6 +76,14 @@ public UserResponseDto editUser(UUID token, Long userId, UserPatchRequestDto use
         }
     }
 
+    @Override
+    public User findByUsername(String username) {
+        if (userRepository.findByUsername(username).isPresent()) {
+            return userRepository.findByUsername(username).get();
+        }
+        throw new com.gradproject2019.auth.exception.InvalidCredentialsException();
```

Comment:
check the import path

Also you should avoid getting this same entity twice, do something like
`Optional<User> user = userRepository.findByUsername(username)`, then do the ifPresent, and Get on this object, not a new one you'll have to get from elsewhere